docs/technical_reference/unit_models/zero_order_unit_models/automate_rst_file_wt3.py

```diff
@@ -279,6 +273,24 @@ def grab_unit_components_feed(unit_class):
 p_subtype_exceptions = {"MetabZO": "hydrogen"}
 has_subtype = {}
 
+# Check that all ZO unit model .py files have an associated .rst file
+ZO_unit_path = Path(zo.__path__[0])
+ZO_unit_list = [
+    path.stem for path in ZO_unit_path.glob("*.py") if path.name not in {"__init__.py"}
+]
+rst_path = Path(__file__).parent
+rst_list = [
+    path.stem for path in rst_path.glob("*.rst") if path.name not in {"index.rst"}
+]
+# print(f"List of zo unit models: {ZO_unit_list}")
+# print(f"List of rst files: {rst_list}")
+
+# Check if lists are equal
+if sorted(ZO_unit_list) == sorted(rst_list):
+    print("Documentation has successfully been created for all ZO unit models")
+else:
+    difference = sorted(list(set(ZO_unit_list) - set(rst_list)))
+    print(f"The following ZO unit model files do not have an rst file: {difference}")
```

Comment:
I see - I'll work on creating this test file then.

Reply:
One thing I should mention (speaking from experience from a few different projects) is that setting up this type of test (i.e. spanning code and documentation) can be quite tricky. I'm happy to discuss more in detail, but in general, we'd want to make sure this test works reliably before adding it to the test suite. Depending on the planned timeline for this feature, it might make sense to consider introducing the test in a separate PR.

Reply:
I think this PR does plenty as-is. Let's leave the test for a follow-up, where we can validate it extensively and separately both locally and on GHA.

Reply:
@bknueven @lbianchi-lbl might have some additional comments regarding this, but you could check out the IDAES wiki for more info on testing: https://github.com/IDAES/idaes-pse/wiki/Testing-Standards

I agree that pushing the test to a follow-up PR would be fine. Then, we could think about any additional complexities that we might want to account for in that test.